diffrax/brownian/base.py

```diff
@@ -1,14 +1,49 @@
 import abc
+from typing import Optional, Union
 
-from ..custom_types import Array, PyTree, Scalar
+import equinox as eqx
+import jax
+from jax import tree_util as jtu
+
+from ..custom_types import Array, LevyVal, PyTree, Scalar
 from ..path import AbstractPath
 
 
+def _levy_tree_transpose(tree_shape, spacetime_levyarea, tree):
+    """Helper that takes a PyTree of LevyVals and transposes
+    into a LevyVal of PyTrees.
+
+    Args:
+        tree_shape:
+        spacetime_levyarea:
+        tree:
+
+    Returns:
+
+    """
+    hh_default_val = 0.0 if spacetime_levyarea else None
```

Comment:
Should this maybe just always be a `None`?

Reply:
I don't think so, it should match the inner tree structure. And there is a leaf here (e.g. 0), whereas a None has no leaves. If I understand correctly at least? I could be wrong, you're the JAX expert.